src/Fabulous.AST/Fabulous.AST.fsproj

```diff
@@ -18,6 +18,7 @@
         <Compile Include="Widgets\ModuleOrNamespace.fs" />
         <Compile Include="Widgets\Oak.fs" />
         <Compile Include="Widgets\Let.fs" />
+        <Compile Include="Widgets\NestedModule.fs" />
```

Comment:
Each file should have its test counterpart?

Reply:
Yeah that is a good idea 👍🏻